packages/api/src/controllers/taskController.js

```diff
@@ -386,15 +388,16 @@ const taskController = {
   async completeHarvestTask(req, res) {
     try {
       const nonModifiable = getNonModifiable('harvest_task');
+      const { user_id } = req.headers;
```

Comment:
Please use `req.user.user_id` instead of `req.headers.user_id`. Honestly, I'm just passing that along from past team members; I *suspect* I know why.